CMSgov/mint-app
# [EASI-3944] Notifications - On Discussion Replied Notification

# EASI-3944

## Changes and Description

* Created new notification meta structure
* Updated and generated GQL
* Updated plan discussion routing to gate sending reply emails based on preferences
* Updated plan discussion to send reply notifications based on preferences



## How to test this change

* Start the application
* Using Postman:
  * Create a default model plan
  * Create a default discussion
  * Create a default discussion reply
  * Query notifications
* Assert postman results are as expected
* Assert unit tests pass as expected


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
